mock/el7-scl.cfg

```diff
@@ -1,7 +1,7 @@
 config_opts['root'] = 'foreman-scl-7-x86_64'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
-config_opts['chroot_setup_cmd'] = 'install @buildsys-build scl-utils-build tfm-build yum vim'
+config_opts['chroot_setup_cmd'] = 'install @buildsys-build scl-utils-build tfm-build yum vim rh-postgresql12-scldevel'
 config_opts['dist'] = 'el7'  # only useful for --resultdir variable subst
 # beware RHEL use 7Server or 7Client
```

Comment:
we probably don't want to update this file

Reply:
oh that was just for local testing, my bad.